packages/nl-design-system-unstable/__docs__/form-patterns.md

```diff
@@ -1,139 +1,270 @@
+
+Indien het onduidelijk is waarom informatie gevraagd wordt informeer bezoekers dan over de noodzaak hiervan maar ook wat de voordelen voor hen zijn.
+
+Voorkom dat gebruikers dezelfde informatie meerdere keren moeten invoeren.
+
+### Tekst labels
+
+Labels zijn de vragen die je een gebruiker stelt. Hier ga je een dialoog aan, dus zorg dat deze zo duidelijk en vriendelijk maar tevens beknopt mogelijk zijn. Laat een bezoeker de vragen makkelijk scannen.
+
+Een goed label wordt buiten een invoerveld geplaatst, niet daarbinnen (de zg. _‘floating labels’_). Op deze manier heeft het label altijd een vaste positie en kan het in een leesbare grootte worden weergegeven. Een label dat in een invoerveld is geplaatst, kan een bezoeker in verwarring brengen, aangezien het voor reeds ingevoerde tekst aangezien kan worden. De leesbaarheid lijdt er ook onder omdat het contrast tussen tekst voorgrond- en achtergrondkleur vaak te laag is. Het kan ook verward worden met inactieve of placeholder tekst. Bovendien kan het ook nodig zijn om soms daadwerkelijke placeholder tekst te tonen.
+
+Een bijkomend voordeel van het plaatsen van een label buiten het invoerveld is dat het klik- en tap-oppervlakte automatisch groter is, waardoor het selecteren van een invoerveld voor zowel muis- als touchscreen gebruikers makkelijker is.
+
+#### Positie van labels
+
+Overweeg om labels links boven een invoerveld te plaatsen. Dit biedt meerdere voordelen die bijdragen aan de gebruiksvriendelijkheid.
+
+* Het biedt voldoende ruimte voor labels van verschillende lengtes. Denk bv. ook aan het vertalen van labels (LTR en RTL talen) door vertaalsoftware.
+* Gebruikers die wensen in te zoomen hebben zo de meeste kans om het label corresponderend met de invoer te zien.
+* Het biedt een natuurlijke ‘scan-lijn’ waardoor een formulier automatisch overzichtelijk en consistent wordt.
+
+### Invoertypes
+
+Voor het beantwoorden van vragen bestaan meerdere elementen:
+
+* Tekst invoer
+* Checkboxen
+* Radio buttons
+* Drop-down menu’s
+* Textarea’s
+* Meervoudige keuzelijsten
+* Recaptcha veld
+
+Al deze invoerelementen dienen vergezeld te gaan van een tekst label dat duidelijk de vraag stelt die beantwoord dient te worden.
+
+De verschillende vormen van invoer elementen en hun specifieke richtlijnen vindt je in de componenten-bibliotheek.
+
+Voor ingevulde/al bekende informatie
+
+Als informatie vooraf al bekend is, bijvoorbeeld een adres of telefoonnummer, vul deze dan in maar laat de bezoeker deze informatie wel wijzigen.
+
+Dit kan op twee manieren:
+
+1. Het vooraf invullen van een formulierveld. Hierdoor heeft een gebruiker de mogelijkheid deze informatie direct aan te passen. Dit is uiteraard alleen van toepassing als de bezoeker is ingelogd.
+2. Het tonen van de informatie (_‘read-only’_) met een functie om deze te wijzigen na het klikken op een button.
```

Comment:
Er is ook nog het [`autocomplete` attribuut](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) in HTML, dat browsers in staat stelt om (niet ingelogde) gebruikers te helpen om bepaalde types informatie (adres, betaalgegevens) in te vullen.